app/assets/javascripts/hydra-editor/keyboard_support.js

```diff
@@ -1,14 +0,0 @@
-Blacklight.onLoad(function() {
-  $('body').on('keypress', '.multi-text-field', function(event) {
-    var $activeField = $(event.target).parents('.field-wrapper'),
-        $activeFieldControls = $activeField.children('.field-controls'),
-        $addControl = $activeFieldControls.children('.add'),
-        $removeControl = $activeFieldControls.children('.remove');
-    if (event.keyCode == 13) {
-      event.preventDefault();
-      $addControl.click()
-      $removeControl.click()
-    }
-  });
-});
-
```

Comment:
Agreed. If you can keep your hands on the keys, why wouldn't you want to keep them there. And holy crow our forms are long! We might be able to alleviate that a little bit by adding tabindex values to jump over optional fields, but that might create additional additional problems. Or just redesign the forms. :-)

Let me double check on the spacebar/enter key usage and then I think we're good to go.